lighthouse-core/config/index.js

```diff
@@ -171,16 +172,79 @@ function filterAudits(audits, auditWhitelist) {
   return filteredAudits;
 }
 
-function expandAudits(audits) {
+function expandAudits(audits, paths) {
+  const rootPath = path.join(__dirname, '../../');
```

Comment:
yeah, could just check in `Runner.getAuditList()`, if not found there, then go through the path search